pkg/graph/resolvers/trb_admin_note.go

```diff
@@ -229,41 +229,6 @@ func GetTRBAdminNoteCategorySpecificData(ctx context.Context, store *storage.Sto
 	return nil, apperrors.NewInvalidEnumError(fmt.Errorf("admin note has an unrecognized category"), note.Category, "TRBAdminNoteCategory")
 }
 
-// UpdateTRBAdminNote handles general updates to a TRB admin note, without handling category-specific data
-// If updating admin notes requires handling category-specific data, see note on UpdateTRBAdminNoteInput in GraphQL schema;
-// break this up into separate resolvers
-// Also, if updating with category-specific data allows changing a note's category, the resolvers will need to null out any previous category-specific data;
-// as well as updating the fields on the admin note record, many-to-many links to documents/recommendations may need to be deleted
-// (which would require implementing storage methods to delete those records)
-func UpdateTRBAdminNote(ctx context.Context, store *storage.Store, input map[string]interface{}) (*models.TRBAdminNote, error) {
```

Comment:
I see this uses the `map[string]interface`, but I assume the resolver itself is unused and so the whole thing is being deleted, right?

Reply:
Yes! I can definitely put back the store method if needed - I know it's a bit out of scope for this PR. I did notice though in the UI that it does not appear to be possible to update notes, is this something that used to be possible?

Reply:
No worries, I just got confused when I saw all of the TRB admin note code being deleted instead of type def changes for gqlgen.

If that resolver is not being used it's probably OK to remove it and the store method, but maybe @ClayBenson94 can weigh in.

Reply:
I think this is fine to delete. It's not called in the UI anywhere, and since introducing the ["category specific" data](https://github.com/CMSgov/easi-app/blob/c9cf9f9f4b401cb16f7a7c7efb5d7292ab298009/pkg/graph/schema.graphql#L2816-L2826), this update query doesn't even really update most of those fields anyways.

Let's 🪓 it!